bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -39,11 +39,27 @@
 	<channel-type id="controlsetpoint">
 		<item-type>Number:Temperature</item-type>
 		<label>Control Setpoint</label>
-		<description>Central heating water setpoint</description>
+		<description>Central heating water setpoint set at boiler</description>
 		<category>Temperature</category>
 		<state readOnly="true" pattern="%.1f %unit%"></state>
 	</channel-type>
 
+	<channel-type id="controlsetpointrequested">
+		<item-type>Number:Temperature</item-type>
+		<label>Control Setpoint Requested</label>
+		<description>Central heating water setpoint requested by Thermostat</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"></state>
```

Comment:
```suggestion
		<state readOnly="true" pattern="%.1f %unit%" />
```

Reply:
I'd followed the pattern of the existing channels in the file, but I've reformatted them all now, and this was a good prompt for me to fix the Control Setpoint Override definition https://github.com/openhab/openhab-addons/pull/9206/commits/c47f416b576fb27ff7e93f1a5c929c1f7a601fbf#diff-e2a0a39908dbee74cd599d47152756cafe01c6a3de9c5067f6fb700da720cd99R60